ts/Data/Stores/DataStore.ts

```diff
@@ -342,6 +326,20 @@ abstract class DataStore implements DataEvent.Emitter {
         return addEvent(this, type, callback);
     }
 
+    /**
+     * The default save method, which fires the `afterSave` event
+     *
+     * @return {Promise<DataStore>}
+     * The saved store.
+     *
+     * @emits DataStore#afterSave
+     * @emits DataStore#saveError
+     */
+    public save(): DataPromise<this> {
+        fireEvent(this, 'saveError', { table: this.table });
+        return DataPromise.reject(new Error('Not implemented'));
```

Comment:
Im not pretty sure why it is not resolve here, but reject

Reply:
Because we can not keep it an abstract class. So it becomes a regular class with some implementations missing. They get eventually overwritten by Store implementations.

Reply:
Sorry, it is still an abstract class, but the default should be an error, when not implemented by the Store.